### ISSUE TITLE
ASAP Due Bug: Re-Review the Binning of Encounters for Time Assumptions

### ISSUE THREAD

**Issue by lzim** [lzim/teampsd#2851]
### Discovered wk4 jan_epic vapor
@lzim 
@emilymetcalf114 
@epearman 

**Add to quant_operation worfklow**

![image](https://user-images.githubusercontent.com/7454688/214442972-d23fda97-3b94-4759-a5d5-45347e4c3424.png)

**Comment by lzim** [comment]
### Considerations for Medication Management
- 30-minutes

1. Reviewed data of the distribution and concluded that **30-minutes was median** (splits the distribution)/mode (most frequent).

2. We **asked**, talking to several program, policy & providers - and concluded most valid across most settings. 
**_GMB Hexagons_ - Top 9 priorites (Zimmerman et al., 2016 manuscript)**
- **_Actual time_** - e.g., PSY (at 60-min) if you provide 90833 (45-min), 90834 (50-min), 90837 (60-min); Prolong Exposure for PTSD was validated internationally as a gold standard treatment, w/90 minute session. 
- [ ] Center for Medicaid & Medicare Services (CMS) phased out the "Add-on" code that everyone used for this, this year.
- **_VA data time_** - Compliance with coding rules, business rules, chartable work.

3. **Simpler** to work with to identify new heuristics - Some forms are precision is not useful for making decisions.


![image](https://user-images.githubusercontent.com/7454688/214443769-1e876e61-b7ea-4c96-addc-e1d31544c29f.png)


### Team (AGG) Care - Data Array of Services**
**_Math applying transforming the units from counts of services into hours of services._** 

Intake
Psy 
EBPsy - 
CC - MHTC f/u
MM - different intervals mediation class
Group - near weekly w/8 Veterans group
Adjunct - No standard episode of care (evidence-based course of engagement initation/dose) applies.

**Comment by dlounsbu** [comment]
**wk5 2024_05 #systems_sme_workflow** 
@lzim 
@dlkibbe 
@jamesmrollins 
@dlounsbu 

**#support_workflow**
@ljmoody 
@lijenn 
@lzim 
We identified that this documentation about binning of encounters was not assigned or documented. We need this included in the [_MTL 3.7 Consult Manual_]([mtl.how/manual])

**Comment by lzim** [comment]
****wk5 2024_05 #support_workflow** - Key for interpreting Appt supply in hours in Team (AGG) Care**
@lzim 
@lijenn 
@ljmoody 

**Data UI works for Team (AGG) Care (all the modules) - Translates counts of grouped CPT-coded encounters (that are defined by time).**

How we decided: 

**Participatory process over 2014-2019 (5 years) - e.g., Medication Management (10 minutes to 40 minutes)** 
- 30-minutes is most realistic if you're thinking in actual time, not just coded encounters.
- Review distributions (mean, median, mode) and ask. 

- Step 1: Counts the number of encounters of that type. 
- Step 2: Vensim DSS processing of those counts it applies the duration of the visit (0.5 as the reflection of the CPT codes for 10-40 minutes [above])

**Comment by lzim** [comment]
@lijenn 
@ljmoody 

Looks like this card hasn't moved moved for a few weeks. 
_Is it because we're unclear about next steps?_
_Should this be added to our agenda for **9am_workflow leads wk3 2024_07**?_

**Comment by ljmoody** [comment]
@lzim, 

- Where would you like this information to be located within the manual?
- What information would you like to be included, exactly? After reviewing the card, we thought the reasoning given in your May 29 comment might be what you would like included; however, were not 100% sure on its meaning. Is there another way the binning of encounters could be described so that both SMEs and laypeople can easily understand?

**Comment by lzim** [comment]
Good question @ljmoody 
_Will you please add this to **wk2 2024_08 #support_workflow** agenda when we're all in the office together again?_ 
- I see it as okay to wait until then.
Thx!

**Comment by lijenn** [comment]
Needs to be discussed wk2 2024_08 support, systems_sme, sim_ui sync workflow.

**Comment by matthewtomo** [comment]
Hello @lzim @jamesmrollins @TomRust,

- I noticed that this card was the only card remaining open for the October sprint (2024_10 for lzim/teampsd).
- Do any of you know what next steps are required for this card?
- Should I add it to an upcoming agenda?

**Comment by lzim** [comment]
@matthewtomo 

Yes, let's add this to an upcoming agenda if not cleared up on the board. 
- [ ] 1. We need to confirm the way that encounter data from the Data UI are translated to appointment supply in the Vensim model files and therefore the Sim UI.
- [ ] 2. Then we need to make sure _MTL_ documentation handles covers this.

Here's two examples I'm aware of off the top of my head without checking in which what we believed were conservative decisions when simulated over the 2-year simulation run. It accounts for what providers called the difference between "actual time" as in time in the day vs. "billable time" as in a patient/provider encounter occuring. 
Of course, these can be adjusted up/down in the Sim UI to move away from these defaults.
1. **Psychotherapy** encounters are treated as 60 minutes in the Vensim model/Sim UI when they are coded as 45-minutes or 60-minutes using CPT coded encounters in the patients chart/CDW/Data UI. The rationale was to have a more realistic and therefore more accurate simulation result because providers cannot see another patient in the 15-minute difference. 
2. **Medication Management** encounters are treated as 30 minutes in the Vensim model/Sim UI when they are coded as 30-minutes or less for a similar reason.

**Comment by matthewtomo** [comment]
**Discussed Monday Ad_Hoc wk1 2024_12**
@matthewtomo @ljmoody 

- @lzim It appears that @jamesmrollins and @TomRust might have a deeper understanding of the questions being asked here on the card.
- @TomRust @jamesmrollins, would the two of you be able to join us on Wednesday for sim_ui_workflow meeting at 3PM EST?

**Comment by matthewtomo** [comment]
**Discussed sim_ui_workflow wk1 2024_12**
@matthewtomo @ljmoody @lzim 

- [ ] @matthewtomo will investigate the Appointment Supply variable for each Vensim Model to gather clarifying information on how  that variable is being handled and relay that information here

**Comment by lzim** [comment]
Thanks @matthewtomo 

- [ ] 1. Open Vensim DSS model file select Appointment Supplye.
- [ ] 2. Select the f(x) equation tool to review the comments for details.

What we expect to see and document is that is a count of encounters multiplied by the visit duration used in the simulation.
e.g. ,A count of 421 Medication Management Encounters * 0.5 (for a half-hour) = 210.5 hours/week in Team Care or 210.5 appts/week in MM.

**Comment by matthewtomo** [comment]
# Notes in each model file for "appointment supply"
- Below are the documentation/notes for each of the model files.
- PSY specifically noted the 60 minute appointments
- For Team Care, I found several "appointment supply" related variables but was not able to find any information on appointment times
- For each of the other models, I checked all "appointment supply" related variables and did not find any other information on appointment times

## Team Care
- Variable name: **Data Appointment Supply (Total)**
![image](https://github.com/user-attachments/assets/7ff04042-8984-4acf-8e19-442bc2677459)
- Team Care also contained versions of two following variables for each of these categories **Adjunct, CC, EBPsy, Group, Intake, MM, and PSY**:
  - "Data Appointment Supply (median) -> **{Category Name}**"
    - The number of appointments available with this team for each service each week. The estimate is calculated using the volume of visits completed each week with the team in each service over one year. (hrs/wk)
  - User **{Category Name}** Appointment Supply
    - The number of official appointments offered each week. This value replaces the default value estimated from the team's data, with the assumption that any increase in supply comes from reducing appointments offered in other services. (hrs/wk)

## CC
- Variable name: **Official Appointment Supply**
  - The number of official appointments offered each week.
![image](https://github.com/user-attachments/assets/3d9ab3aa-2083-499e-966d-60e023aa8374)


## MM
- Variable name: **Official Total Appointment Supply**
  - The total number of official Medication Management appointments offered by the team each week.
- No other Appointment supply related variables provided any further insight
![image](https://github.com/user-attachments/assets/ef4cde4d-c652-4c43-9879-642115222a2e)


## PSY
- Variable name: **Appointment Supply**
  - Total number of 60 minute Psychotherapy appointments offered by the team each wk.
![image](https://github.com/user-attachments/assets/0285ea9a-49b6-4b75-8c5e-f3e4123b1631)


## Team Flow
- Could not find a variable associated with appointments
![image](https://github.com/user-attachments/assets/aeb65103-e921-4dc9-b499-70d2e1d86cb7)


fyi: @lzim

**Comment by lzim** [comment]
@TomRust - I know we're busy with **#mtl_hawaii** but, _can you tell what we're trying to confirm on this card and help us out?_

**Comment by matthewtomo** [comment]
@lzim @TomRust with the information and screenshots I provided above, are we able to complete the first bullet point of remaining tasking for this card? If not, is there more information you would like for me to provide?

> - [ ] 1. We need to confirm the way that encounter data from the Data UI are translated to appointment supply in the Vensim model files and therefore the Sim UI.
>  - [ ]  2. Then we need to make sure MTL documentation covers or explains this.

Once point 1 is completed, point 2 seems to be related to card: #3244

**Comment by lijenn** [comment]
@matthewtomo, @TomRust, @lzim, @jamesmrollins is there anything else we need to do for this card? It's the only card preventing us from closing the 2024_10 epic card #3180. Thanks!
